### DISCUSSION TITLE
Scrum Prep Milestone 3 - Week 3

### DISCUSSION THREAD

**Discussion by Ar2691** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#177] [Scrum Preps]
**Nov 11 - Nov 15**

**Comment by cristianMarcial** [comment]
Multi-Language Support Team Leader Here. on this week we created the parser which reads the strings on the document which holds the UI content strings of the app, and allocates them in a dictionary which can be accessed by all the screen modules, such as Profile, College Planning, Main Menu, etc.

**Comment by Gian-Go** [comment]
Added the team's Agile Development Structure description in the documentation. Will be working on further formatting changes and a new Petri Net Diagram for the documentation during the weekend.

**Comment by keisanabria** [comment]
Other than this progress: ![53FBE22E-0C4E-45F2-81D8-6266D5150F62](https://github.com/user-attachments/assets/8454dcca-c707-497e-ba51-c00fe6e337f4)

I haven't made more progress.

**Comment by nicolle-feliciano** [comment]
Made research on Lecture Topic Tasks that might benefit the project. Will be creating them during the weekend and working on the documentation.

**Comment by carlospepin23** [comment]
I completed the task #175 to benefit the semester college planning team.

**Comment by PedroRodz** [comment]
Completed a task and opened two more that are currently in progress

**Comment by Yamil-Serrano** [comment]
Participate in the creation of the parser for Multi-Language Support Team.

**Comment by christianmedina-dev** [comment]
Had a short meeting with the Registration Team to coordinate our next Tasks and workload. Will start working on it this week.

**Comment by Carloscaban** [comment]
I could not work on any task, had a hell week. But at least the team assigned tasks.
